data/data/bootstrap/files/usr/local/bin/kubelet.sh.template

```diff
@@ -9,6 +9,9 @@
     --container-runtime=remote \
     --container-runtime-endpoint=/var/run/crio/crio.sock \
     --runtime-request-timeout="${KUBELET_RUNTIME_REQUEST_TIMEOUT}" \
+{{- if .APIVIPIPv6 }}
```

Comment:
Would it make sense to make the name of this variable more specific to its use rather than specific to its source?
```suggestion
{{- if .UseIPv6ForNodeIP }}
```